src/Build.UnitTests/ProjectCache/ProjectCacheTests.cs

```diff
@@ -1349,6 +1351,86 @@ Task<BuildResult> BuildProjectFileAsync(int projectNumber)
             }
         }
 
+        [Theory]
+        [InlineData(false, false)]
+        // TODO: Reenable when this gets into the main branch.
+        //[InlineData(true, true)]
```

Comment:
When what gets into which main branch?

Reply:
Literal meaning. When this comment gets into the main branch, the test should be reenabled :)

Reply:
ah, so there's a fix in our main that's not in our 16.11 that means we should change this on merge?

Reply:
Correct, the fix is in https://github.com/dotnet/msbuild/pull/6400 which we deemed to risky for 16.11